src/components/AddTransaction/index.js

```diff
@@ -99,6 +97,28 @@ export default function AddTransaction() {
 						);
 					})}
 				</select>
+				<input
+					id="income"
+					checked={transaction.change > 0}
+					type="radio"
+					value="income"
+					name="transactionType"
+					onChange={() => {
+						setTransaction({...transaction, change: transaction.change * -1});
+					}}
+				/>
+				<label htmlFor="income">Income</label>
+				<input
+					id="expense"
+					checked={transaction.change < 0}
+					type="radio"
+					value="expense"
+					name="transactionType"
+					onChange={() => {
+						setTransaction({...transaction, change: transaction.change * -1});
+					}}
```

Comment:
Is it correct that both `expense` and `income` have the same transaction change function? 🤔 I am talking about this line:
```js
setTransaction({...transaction, change: transaction.change * -1});
```

Reply:
Maybe with a good name? :D

Reply:
Yep, will do! 👩‍💻